src/mon/MonClient.cc

```diff
@@ -855,6 +855,29 @@ bool MonClient::ms_handle_reset(Connection *con)
   }
 }
 
+bool MonClient::ms_handle_throttle(ms_throttle_t ttype) {
+  switch (ttype) {
+  case ms_throttle_t::MESSAGE:
+    break; // TODO
+  case ms_throttle_t::BYTES:
+    break; // TODO
+  case ms_throttle_t::DISPATCH_QUEUE:
+    {
+      //cluster log a warning that Dispatch Queue Throttle Limit hit
+      if (!log_client) {
+        return false; //cannot handle if the daemon didn't setup a log_client for me
+      }
+      LogChannelRef clog = log_client->create_channel(CLOG_CHANNEL_CLUSTER);
```

Comment:
Create this channel at startup.

Reply:
The `log_client` is `NULL` since the MonClient is created. But still we could move it to `MonClient::set_log_client()` (and make clog as a member variable?). That's where MonClient knows that a Daemon had set-up the `log_client` object for it. 

But that's irrelevant because we could call log_client->create_channel() many times, which is okay. If the channel exists, it would return the existing channel. Please see LogClient::create_channel().

Reply:
Alright.

Reply:
create_channel() doesn't seem to prevent invalid channel names though. Looks like it would create a channel named `xyz` :)